### DISCUSSION TITLE
"Rich text" for emails

### DISCUSSION THREAD

**Discussion by wassson** [caley-io/email-rails#78] [Q&A]
Typically an email platform will provide complex, rich text-like formatting features. How should we go about solving this problem in a Rails-specific way? Rails already has ActionText and Trix. Should we pursue this kind of solution, custom, or another route? How have you solved or seen this problem solved in the past?

**Comment by sriram2520** [comment]
[Quill](https://quilljs.com/) is another popular library. But my vote would be to go with ActionText column + Trix, simply for the reason that basecamp uses it and seems to be the choice of the rails community these days. So we can expect any update on trix considers compatibility with rails + actiontext + stimulus before doing major changes.

**Reply by wassson** [reply]
I think that's the way I'm leaning for now. From a UI perspective, how flexible is Trix? Would we have complete control. I have yet to actually use it in a project of any size
